src/gsad.c

```diff
@@ -3828,6 +3833,120 @@ gsad_add_content_type_header (struct MHD_Response *response,
     }
 }
 
+/**
+ * @brief Add all local IP addresses to a GHashTable.
```

Comment:
This doesn't include localhost, 127.x.x.x and the ipv6 local addresses right? So the description is a bit confusing.
Maybe move adding the localhost addresses to the hashmap also into this function.